pkg/systemd/logDetails.jsx

```diff
@@ -131,14 +131,14 @@ export class LogEntry extends React.Component {
                         }
                     });
 
-                    this.setState({ entry: entry, loading: false, error: "", problemPath: path, abrtService: service });
+                    this.setState({ entry, loading: false, error: "", problemPath: path, abrtService: service });
                 })
-                .catch(err => this.setState({ entry: entry, loading: false, error: err.toString() }));
+                .catch(err => this.setState({ entry, loading: false, error: err.toString() }));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-173310-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)